src/CommandingOfficers/Venge.java

```diff
@@ -237,6 +238,14 @@ protected void revert(MapMaster gameMap)
       }
       boostedUnits.clear();
     }
+
+    @Override
+    public GameEventQueue getRevertEvents(MapMaster gameMap)
```

Comment:
Did this part of Ironwill just not work before?

Reply:
It worked just fine. It was in `revert()` rather than actually pushing out an event.

Reply:
Well, it looks like that is still there. Except it heals them because `IRONWILL_WOUND` is now positive. An 8 HP unit which gets iron willed will still be at 8 afterwards.